controllers/backup.go

```diff
@@ -104,7 +102,7 @@ func (r *ClusterGroupUpgradeReconciler) triggerBackup(ctx context.Context, clust
 
 // backupPreparing handles conditions in BackupStatePreparingToStart
 // returns: error
-func (r *ClusterGroupUpgradeReconciler) BackupPreparing(ctx context.Context, clusterGroupUpgrade *ranv1alpha1.ClusterGroupUpgrade,
+func (r *ClusterGroupUpgradeReconciler) backupPreparing(ctx context.Context, clusterGroupUpgrade *ranv1alpha1.ClusterGroupUpgrade,
```

Comment:
Are you switching this to non capital start as it's going to be used only as an internal helper?

Reply:
yes, this function is exclusively used for the backup.